### DISCUSSION TITLE
What's easiest way to factory reset to default settings overall? Generated images not display after finish rendering.

### DISCUSSION THREAD

**Discussion by nnqq2603** [vladmandic/sdnext#581] [Q&A] [answered]
I tried this fork, on the 1st time I run it, things were fine then I did some change in settings, tweak some path here and there. And then next run it can't display generated image in UI (I sucessfully generated and saved them to disk but doesn't display in UI). And to be honest, I don't remember what exactly changed because it's more than one or two things were changed.

So what's fastest/easiest way to reset the program to default state? I tried remove whole **venv** folder (which was one of ways I did in auto1111 and got a lot of errors solved that way) and let it redownload but still same issue. Other than just delete the whole vlad StableDiffsuion folder and reinstall all from scratch is there less destruction way?

**Comment by vladmandic** [comment, marked as answer]
Just delete config.json and UI-config.json and on the next start they will be recreated with defaults.

**Reply by nnqq2603** [reply]
Thanks, that solved it. Images displays well now.
